projects/safety_recipes/human_safety_evaluation/format_safety_ready.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import os
+import json
+import argparse
+
+from parlai.utils.io import PathManager
+import parlai.utils.logging as logging
+
+
+"""
+python projects/safety_recipes/human_safety_evaluation/format_safety_ready.py --world-logs-path tmp/world_logs.jsonl --eval-logs-dir tmp/human_safety_evaluation
+"""
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--world-logs-path",
+        type=str,
+        help="path to the world_logs.jsonl generated from parlai evaluate_model",
+    )
+    parser.add_argument(
+        "--eval-logs-dir",
+        type=str,
+        help="dir to the store annotate_indices.jsonl and task_data.jsonl ready for direct use in human safety evaluation format",
+    )
+
+    args = parser.parse_args()
+    world_logs_path = args.world_logs_path
+    chatlogs = []
+    with PathManager.open(world_logs_path) as data_file:
+        for l in data_file.readlines():
+            episode = json.loads(l.strip())
+            # TODO: when conversation format is finished please remove this line;
```

Comment:
what does this mean?

Reply:
It seems like there is a bug in conversation format that would generate lines like:
```
{"dialog": [[{"batch_padding": true, "episode_done": true, "id": "bot_adversarial_dialogue:HumanSafetyEvaluation.persona_False_flatten_False"}, {"id": "TransformerGenerator", "episode_done": false}]], "context": [], "metadata_path": "tmp/world_logs.metadata"}
```
I added a hack to skip those when parsing but, there is room for removing that hack after the bug above is fixed.